react/src/utils/backendrouter.js

```diff
@@ -0,0 +1,38 @@
+const DEFAULT_BACKEND = "flask"
+const SUPPORTED_BACKEND_TYPES = {
```

Comment:
This config for supported backend types could potentially be put into a config file if we plan to add more backend types. But it seemed like overkill to do that now.